jpl/cli/__init__.py

```diff
@@ -1 +1 @@
-from .main_cli import *
\ No newline at end of file
+from .main_cli import *
```

Comment:
Yeah?

Reply:
What do you suggest in place, this persists cli to user, bespoke imports?

Reply:
What do you mean by that?  What's the need for this import here?  Generally just not a fan of importing `*`.

Reply:
In order to access the CLI, the functions need to be exposed up the call chain thru the __init__

Reply:
Interesting...

Reply:
Resolving with abs imports